src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -0,0 +1,213 @@
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE USER {your username} WITH PASSWORD '{your password}';
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    GRANT SYSTEM REPLICATION TO {your username};
+    ~~~
+
+    If you need to change the password later, refer to [`ALTER USER`]({% link {{ page.version.version }}/alter-user.md %}). {% comment  %}Add link to https://www.cockroachlabs.com/docs/stable/ui-overview#db-console-security-considerations for secure clusters{% endcomment %}
+
+1. (Optional) If you would like to ignore row-level TTL deletes in LDR, set the `ttl_disable_changefeed_replication` storage parameter on the table. On the **source** cluster, alter the table to set the table storage parameter:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER TABLE {table_name} SET (ttl_disable_changefeed_replication = 'true');
+    ~~~
+
+    {% comment  %} Add link to the example for this on the create logical replication stream page once published. {% endcomment %}
+
+## Step 2. Connect from the destination to the source
+
+In this step, you'll set up an external connection from the destination cluster to the source cluster. Depending on how you manage certificates, you must ensure that all nodes between the clusters have access to the certificate of the other cluster.
+
+You can use the `cockroach encode-uri` command to generate a connection string containing a cluster's certificate.
+
+1. On the **source** cluster in a new terminal window, generate a connection string, by passing the replication user, node IP, and port, along with the directory to the source cluster's CA certificate:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach encode-uri {replication user}:{password}@{node IP}:26257 --ca-cert {path to CA certificate} --inline
+    ~~~
+
+    The connection string output contains the source cluster's certificate:
+
+    ~~~
+    {replication user}:{password}@{node IP}:26257?options=-ccluster%3Dsystem&sslinline=true&sslmode=verify-full&sslrootcert=-----BEGIN+CERTIFICATE-----{encoded certificate}-----END+CERTIFICATE-----%0A
+    ~~~
+
+1. In the SQL shell on the **destination** cluster, create an [external connection]({% link {{ page.version.version }}/create-external-connection.md %}) using the source cluster's connection string. Prefix the `postgresql://` scheme to the connection string and replace `{source}` with your external connection name:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE EXTERNAL CONNECTION {source} AS 'postgresql://{replication user}:{password}@{node IP}:26257?options=-ccluster%3Dsystem&sslinline=true&sslmode=verify-full&sslrootcert=-----BEGIN+CERTIFICATE-----{encoded certificate}-----END+CERTIFICATE-----%0A`;
+    ~~~
+
+## Step 3. Start LDR
```

Comment:
ok actually we fixed this restore correctness issue yesterday, so now we can include the cursor bit... So sorry for the whiplast. ARe you able to include the instructions from https://docs.google.com/document/d/1ou6cxZdWiUDC74Zo-dDCTt9IqaDgPf16H3OqdHrwEqo/edit?tab=t.0 for initial bootstrap?

Reply:
I think this might be better placed as an example on the `CREATE LOGICAL REPLICATION STREAM` ref page, as an alternative way to start LDR. BUT, do let me know if you want this to replace what we have in the setup guide right now.